pip/req/req_set.py

```diff
@@ -281,18 +279,14 @@ def prepare_files(self, finder):
                 is_wheel = False
                 if req_to_install.editable:
                     if req_to_install.source_dir is None:
```

Comment:
Well this is a commit from Dec 2014, I'm confident on everything except the removal of:

    if not os.path.exists(self.build_dir):
        _make_build_dir(self.build_dir)

But looking again, it seems editable package do not need one: everything is done via `self.source_dir` and in the end the important variable is `location`.
So in the end, I'm confident again on the uselessness of calling `_make_build_dir(self.build_dir)` :)